config/default.yml

```
@@ -19,6 +19,7 @@ AllCops:
     - 'Gemfile'
   DisplayStyleGuide: true
   DisplayCopNames: true
+  NewCops: true
```

I think this is the suggestion from rubocop:

```suggestion
  NewCops: enable
```